.bazelrc

```diff
@@ -23,7 +23,7 @@ test --test_output=errors
 query --nohost_deps --noimplicit_deps
 
 # Adding in c++11 compiling, this really should be done in the android toolchains
-build --cxxopt="-std=c++11" --host_cxxopt="-std=c++11"
+build --cxxopt="-std=c++17" --host_cxxopt="-std=c++11"
```

Comment:
The rest of our codebase isn't quite ready for this. I'm working on fixing that, will let you know when it's ready.